### ISSUE TITLE
1.7.6.0 Translation only shows very few strings

### ISSUE THREAD

**Issue by johanlantz** [PrestaShop/PrestaShop#14916]
**Describe the bug**
Very few translations actually show.

**To Reproduce**
In my case on 1.7.6.0. 
1. Open the translation tap, select for instance "Theme translation" and then classic and english. 
2. Choose for instance
Shop->Theme->Checkout
Result: I get only 3 strings for translation. There should be plenty more, also no idea why these 3 work.
Expected result: I would like to be able to translate all strings.

I have tried with different themes and languages with no luck. I can translate the strings that show up and that works but most strings are not visible.

I am about to finish a migration from 1.5 to 1.7 and this is the only thing left so. I have tried to find a workaround but with no success so if there is anything I can test I would be happy to do so (I have tried to do an update of the language from the same menu but there was no difference.

I know there are several issues open for Translations in general but this one seems different (that I only see a subset of the strings that should be available for translation).

Any idea or advice is very appreciated. The behaviour is the same on the shop to which I have migrated as well as on a clean 1.7.6 installation running locally in a docket.

For Shop.Themes.Checkout, the only results I get for classic are the 3 items below.
**Screenshots**
![Screenshot 2019-07-30 at 14 47 49](https://user-images.githubusercontent.com/7461655/62130689-d99b6d00-b2d9-11e9-8479-474c96f78135.png)

**Additional information**
PrestaShop version: 1.7.6.0
PHP version: 7.2

**Comment by khouloudbelguith** [comment]
Hi @johanlantz,

I did not manage to reproduce the issue with PS1.7.6.0.
I attached a screen record
https://drive.google.com/file/d/1Kgl9qdp7splgSBDHCl2H1t_N3_Bn-Imp/view
Could you please provide us with more info? We need more details to understand how we can reproduce your issue:
 - host
 - server setup and configuration
 - PrestaShop version (source)
 - debug mode report
 - PHP error logs
 - apache error log
 - javascript console log
 - screenshots

Don't you know how to get this information? Please read the following article:
http://build.prestashop.com/howtos/misc/how-to-create-bug-report/

Thanks!

**Comment by johanlantz** [comment]
Hi @khouloudbelguith 

Thanks for the quick test.

I have no idea why, but the translations just started to show,.

Literally the only thing that has happend is that I managed to translate one own my own modules using the old system. When I came back to theme translation, all the strings were visible. Something must have been reset or cleared or restarted. I will continue to test but right now, they are all visible again and I can successfully translate theme variables.

I am still not able to translate Prestashop modules though, it just says the file does not exist but since this issue was theme related, we can close it for now.
